src/main/resources/easy_words

```diff
@@ -235,11 +235,340 @@ VISIT
 ENTER
 WORTH
 ROMAN
+STEPS
+VIEWS
+SPENT
+SCENE
+BRAIN
+SHAPE
+HAPPY
+OLDER
+TRIAL
+PLANS
+SALES
+USUAL
+PRIOR
+APPLY
+HORSE
+SPEED
+INDEX
+BIRTH
+OUGHT
+NAMED
+ERROR
+JOINT
+FOCUS
+PETER
+GIRLS
+QUEEN
+URBAN
+BOUND
+TESTS
+KINDS
+GREEK
+COAST
+PLATE
+CHECK
+WALLS
+BREAK
+RATIO
+SIDES
+SIGHT
+MEDIA
+BROAD
+AGENT
+AWARE
+YOUTH
+PAGES
+SLEEP
+ITEMS
+FRESH
+KNOWS
+DRIVE
+RURAL
+METAL
+GUIDE
+STEEL
```

Comment:
Words are not really easy, should have used the website linked on discord for easier words, I would suggest move more of these to the hard_words file